tests/HTTP/RequestTest.php

```diff
@@ -35,7 +35,7 @@ public function testGetQueryParametersNoData(): void
     }
 
     /**
-     * @backupGlobals
+     * @backupGlobals enabled
```

Comment:
backupGlobals should explicitly have the keyword "enabled"

Reply:
Did this annotation work as expected in phpunit without the suffix? If not we might just drop it